docs/pages/conferences.md

```diff
@@ -24,6 +24,7 @@ the conference.
 
 | Conference | Location | Proceedings |
 | ---------- | --------- | --------- |
+| [ISMIR 2019](https://ismir2019.{{site.domain}}/) | 4-8 November 2019, Delft (Netherlands) | [Proceedings]({{site.base_url}}/conferences/ismir2019.html) |
```

Comment:
In most material we've been saying "The Netherlands" as a location, but I see that Utrecht (below) just says "Netherlands", so I used it here for consistency. I'm open to changing it